core/crypto/vrf/p256/p256_test.go

```diff
@@ -179,40 +186,37 @@ func TestVRF(t *testing.T) {
 }
 
 func TestProofToHash(t *testing.T) {
```

Comment:
Please add a note about how to regenerate the test data.

Reply:
Done.